### ISSUE TITLE
UI shifts on virtual branch commit

### ISSUE THREAD

**Issue by DamianZaremba** [gitbutlerapp/gitbutler#3188]
Version: 0.10.24 (20240312.205643)

In a project with 8 virtual branches, when committing the "most right" branch, the UI scrolls all the way to the left upon the commit finishing.

This is quite annoying in 2 cases;
- You have started typing in a commit message for another virtual branch, during which the UI scrolls away to the trunk branch, loosing the context that you where typing in
- You wish to push the virtual branch commit to the remote, which requires scrolling all the way back to the right

**Comment by PavelLaptev** [comment]
Thank you. Interesting. I'll try to investigate this. 
Might be it's somehow related to the WebKit version but now sure.
Would be nice if you can test the next release. It will be probably tomorrow.

On Sonoma 14.1 I don't experience this issue.

https://github.com/gitbutlerapp/gitbutler/assets/18498712/db9756c9-c02d-4e35-acf4-c5e5e26d5642

**Comment by PavelLaptev** [comment]
@mtsgrd thank you!

**Comment by DamianZaremba** [comment]
I've actually updated to Sonoma 14.4 and GitButler 0.10.27 since reporting this; currently haven't noticed the issue, but also haven't been dealing with lots of virtual branches.

Perhaps relevant, or perhaps not, I also noticed some odd behaviour when you have a branch e.g. creating a PR or pushing and start working on another branch... it appears like on a state change some refresh action happens which 'jiggles the ui around'.

I'll see if I can make a reproduction and screen capture it during usage.....
